applications/asset_tracker/src/env_sensors/bsec.c

```diff
@@ -368,8 +368,8 @@ void env_sensors_set_send_interval(const uint32_t interval_s)
 	if (data_send_interval_s) {
 		/* restart work for new interval to take effect */
 		env_sensors_poll();
-	} else if (k_delayed_work_remaining_get(&env_sensors_poller) > 0) {
-		k_delayed_work_cancel(&env_sensors_poller);
+	} else if (k_ticks_to_ms_ceil32(k_work_delayable_remaining_get(&env_sensors_poller)) > 0) {
+		k_work_cancel_delayable(&env_sensors_poller);
```

Comment:
What happens if this fails?

Reply:
the work item will return early if data_send_interval_s==0

Reply:
please add a comment recording that so future maintainers don't need to analyze the code to find out.

Reply:
done